dor/service_layer/handlers/catalog_revision.py

```diff
@@ -18,17 +19,19 @@ def catalog_bin(event: PackageStored, uow: AbstractUnitOfWork) -> None:
     literal_common_metadata_path = matching_object_file.literal_path
     common_metadata = json.loads(literal_common_metadata_path.read_text())
 
-    bin = Bin(
+    revision = Revision(
         identifier=event.identifier,
         alternate_identifiers=[root_resource.alternate_identifier.id],
+        revision_number=1,
```

Comment:
We'll need to revisit this when we get to updates, and probably fetch the previous revision to get the next revision number.